src/Microsoft.ML.DataView/DataViewType.cs

```diff
@@ -12,6 +12,11 @@ namespace Microsoft.ML.Data
     /// <summary>
     /// This is the abstract base class for all types in the <see cref="IDataView"/> type system.
     /// </summary>
+    /// <remarks>
+    /// Those that which to utilize the extension of the <see cref="IDataView"/> type system should
```

Comment:
`Those that which to utilize`?

Reply:
Funny way of writing "wish." A regional dialect. :P But I think I can clean up this whole sentence...